src/main/java/com/synopsys/integration/detect/workflow/blackduck/integratedmatching/model/ScanCounts.java

```diff
@@ -0,0 +1,39 @@
+package com.synopsys.integration.detect.workflow.blackduck.integratedmatching.model;
+
+import com.google.gson.annotations.SerializedName;
+import com.synopsys.integration.util.Stringable;
+
+public class ScanCounts extends Stringable {
+
+    @SerializedName("PACKAGE_MANAGER")
+    private int packageManager;
+
+    @SerializedName("SIGNATURE")
+    private int signature;
+
+    @SerializedName("BINARY")
+    private int binary;
+
+    public ScanCounts(final int packageManager, final int signature, final int binary) {
+        this.packageManager = packageManager;
+        this.signature = signature;
+        this.binary = binary;
+    }
+
+    public int getPackageManager() {
+        return packageManager;
+    }
+
+    public int getSignature() {
+        return signature;
+    }
+
+    public int getBinary() {
+        return binary;
+    }
+    
+    @Override
+    public String toString() {
+        return "PACKAGE_MANAGER: " + packageManager + ", SIGNATURE: " + signature;
```

Comment:
Are you purposefully not including binary counts in this summary?

Reply:
Yes, will include back once we support binary scans.